### ISSUE TITLE
"plus emails" bypass email taken check

### ISSUE THREAD

**Issue by searinminecraft** [supertuxkart/stk-addons#166]
If you use an email and add `+whateveryouwant` before the domain (example: `kitakita+whateveryouwant@disroot.org`, it effectively bypasses the checking if the email is taken or not, as it will think its distinct and will proceed with the registration anyway.

![image](https://github.com/supertuxkart/stk-addons/assets/114207889/4b742351-5540-418d-a5eb-88f3c05ae217)
![image](https://github.com/supertuxkart/stk-addons/assets/114207889/fb9b9921-806d-4cd2-aaaa-1b22dbfe89c7)
![image](https://github.com/supertuxkart/stk-addons/assets/114207889/2cb109fb-2536-4697-93dd-b22cc0bbf716)

**Comment by qwertychouskie** [comment]
This is expected behavior, the whole idea of "plus emails" is that you can have unlimited unique email addresses that have all their emails show up in the same inbox.

**Comment by searinminecraft** [comment]
> This is expected behavior, the whole idea of "plus emails" is that you can have unlimited unique email addresses that have all their emails show up in the same inbox.

Yes. The problem here is that people may take advantage of that for spam or abuse.

**Comment by qwertychouskie** [comment]
Any person can create any number of email addresses, plus addresses are just a convenience.

IMHO this is a wontfix.
